Weather App/Models/Weather.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Weather.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 31.08.2021..
+//
+/*
+ 
+ Ikona vremenske prognoze, Trenutna temperatura,
+ Ime lokacije, Minimalna i Maximalna temperatura,
+ Vlažnost zraka, Tlak zraka, te Jačina vjetra
+ */
+
+import Foundation
+
+struct Weather : Codable, Equatable {
```

Comment:
Why is Weather conforming to Equatable?

Reply:
It should not, it stayed for no reason, I will delete it.